<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <arg name="serial_port" default="/dev/ttyUSB2"/>
  
  <!-- This calls the ROS node we have written, using the parameters above -->
  <node name="precision_gripper_server" pkg="o2as_precision_gripper" type="precision_gripper_action_server.py" respawn="false" output="screen">
    <param name="serial_port" value="$(arg serial_port)"/>
    <rosparam file="$(find o2as_precision_gripper)/config/precision_gripper_parameters.yaml" command="load" />
  </node>
</launch>
